src/Tizen.NET.MaterialComponents/Components/MList.cs

```diff
@@ -0,0 +1,78 @@
+﻿using ElmSharp;
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MList : GenList, IColorSchemeComponent
+    {
+        HashSet<GenItem> _realizedItems = new HashSet<GenItem>();
```

Comment:
Do we need to hold additional item list?

Reply:
That's because `GenList` doesn't provide to access to `GenItems` they have internally.
`GenItems` are needed whenever the theme color is changed, `MList`  saves the items when it is realized.